WORKING_MODE.md

```diff
@@ -0,0 +1,12 @@
+# Working mode of the AMP TSC
+
+- We meet every 2 weeks for 1h via video conference.
```

Comment:
Call out specifically that we see this as status quo and something we'd adapt as we see fit